<launch>

  <!-- controller utils -->
  <include file="$(find turtlebot3_manipulation_moveit_config)/launch/move_group.launch"/>

  <node name="q_bot_action" pkg="q_learning_project" type="bot_action.py" respawn="true" output="screen"/>
  <node name="q_movement" pkg="q_learning_project" type="movement.py" respawn="true" output="screen"/>
  <node name="q_vision" pkg="q_learning_project" type="vision.py" respawn="true" output="screen"/>
  <node name="q_arm" pkg="q_learning_project" type="arm_manipulation.py" respawn="true" output="screen"/>
  <node name="q_manipulation_publisher" pkg="q_learning_project" type="manipulation_publisher.py" respawn="false" />

</launch>
